app.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express');
+const app = express();
+
+const methodOverride = require('method-override');
+const bp = require('body-parser');
+const exphbs = require('express-handlebars')
+
+const Products = require('./db/products.js');
+const DS_Prod = new Products();
+
+const Articles = require('./db/articles.js')
+const DS_Articles = new Articles();
+
+const PORT = process.env.PORT || 8000;
+
+app.use(express.static('public'));
+
+app.use(bp.urlencoded({ extended: true }));
+
+app.use(methodOverride('_method'));
+
+app.engine('.hbs', exphbs({ defaultLayout: 'main', extname: '.hbs' }));
+app.set('view engine', '.hbs');
+
+// GET - Home Page
+app.get('/', (req, res) => {
+    // res.send('IZ WORKING?')
+    res.render('home');
+});
+
+// ***** PRODUCTS ***** //
+
+//GET - Create New Products Form
+app.get('/products/new', (req, res) => {
+    res.render('form');
+})
+
+// GET - Products Page
+app.get('/products', (req, res) => {
+    const allProds = DS_Prod.all();
```

Comment:
initiate at top of file, to have it already in cache - and for a get request, use a get all method from db class